db/migrate/20170608082243_add_fields_to_feedback_submission.rb

```diff
@@ -1,4 +1,4 @@
-class AddFieldsToFeedbackSubmission < ActiveRecord::Migration
+class AddFieldsToFeedbackSubmission < ActiveRecord::Migration[4.2]
```

Comment:
We could think about consolidating the migrations so we don't have to keep maintaining very old migration files. To start a new db we would use the schema.